enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -536,6 +540,10 @@ func (p *PebbleCache) deleteOrphanedFiles(quitChan chan struct{}) error {
 			default:
 			}
 
+			dupPartitionPattern := fmt.Sprintf("%s/%s", part.ID, part.ID)
```

Comment:
Could we not use `p.isolateByGroupIDs` here? 
Should be fine to assume that a cache root should have only one type of directory structure.

Reply:
When manually testing I accidentally reused the same directory and set isolateByGroupIDs=true and scanForBrokenFiles deleted everything in the cache... So I added this as a precaution. This code should be temporary anyway and will be deleted after the migration is complete

Reply:
Actually realized that we won't be able to clean up this code, because some on-prem customers may still be on the old naming even after the cloud migration. That being said, I still think the precaution is valuable in case someone manually deploys an old config on dev or something to do testing without realizing the repercussions of reusing the directory

Reply:
AFAIK, nobody is using pebble on-prem yet.

Reply:
Oh great - then we'll be able to clean this up